README.md

```diff
@@ -181,21 +187,24 @@ import React from 'react'
 function App() {
   return (
     <ApiReferenceReact
-      configuration={{
-        spec: {
-          url: 'https://petstore3.swagger.io/api/v3/openapi.json',
-        },
-      }}
-    />
-  )
+      configuration = {
```

Comment:
I think I’d prefer this to be intended. Wdyt?

Reply:
I reverted it :D